info/README.md

```diff
@@ -0,0 +1,7 @@
+UPDATE MAIN README
+
+* To update the codestyle text in the main README, you **NEED TO CHANGE** the content of the file `guide-chapter-<N>.md`, the corresponding section of the rules that you changed / added.
+
+* After the updates are made **NEED TO PERFORM GENERATION** using gradle `generateFullDoc` and `generateAvailableRules` contained in `/IdeaProjects/diktat/info/build.gradle.kts` file. Example - ` . gradlew :generateFullDoc --continuous` and ` . gradlew :generateAvailableRules --continuous`.
```

Comment:
```suggestion
* After the updates are made **NEED TO PERFORM GENERATION** using gradle `generateFullDoc` and `generateAvailableRules` contained in `info/build.gradle.kts` file. Example - `./gradlew :generateFullDoc` and `./gradlew :generateAvailableRules`.
```
Usually paths in repo are considered relative to the repo root. Also, the command `. gradlew` is very different from `./gradlew`. Btw, why `--continuos`? It shouldn't have any effect on these tasks.

Reply:
Done [39224da](https://github.com/saveourtool/diktat/pull/1384/commits/39224da3b5f978bedef32554403a66973e17e52d)